code/modules/cargo/materials_market.dm

```diff
@@ -123,11 +138,14 @@
 	if(HAS_TRAIT(SSeconomy, TRAIT_MARKET_CRASHING))
 		market_crashing = TRUE
 
+
+
 	data["catastrophe"] = market_crashing
 	data["materials"] = material_data
 	data["creditBalance"] = balance
 	data["orderingPrive"] = ordering_private
 	data["canOrderCargo"] = can_buy_via_budget
+	data["currentOrderQuantity"] = order_quantity
```

Comment:
I already did this. Please review #79145 before making any more changes

Reply:
As for your specific UI tweaks i can in co-operate them into my PR, no problem

Reply:
I see, the scope of your PR greatly increased while I was away for work.